### PR TITLE
added 12 files for week1-js2 homework

### PR DESCRIPTION
**Pull request by TITHI1244**

### CODE REVIEW

**Review by Rassibassi — COMMENTED** [review]
Good work, let me know if you have any questions regarding my comments

File: javascript/javascript2/week1/shortest.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+const danishWords = ["bil", "plante", "kaffe", "bog", "ø", "planetarium"];
+
+function findTheShortestWord(inputArray) {
+    let shortestWord = inputArray[0];
+    for(let word of inputArray) {
```
[the remark this answers is not in the document]
Comment (TITHI1244) [reply javascript/javascript2/week1/shortest.js:5]: Yes, thanks. I did not see that :)

File: javascript/javascript2/week1/count_letters.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
+function countDanishLetters(inputString) {
+    let countÆ = 0;
+    let countØ = 0;
+    let countÅ = 0;
+    for(let letter of inputString) {
+        if(letter.toLowerCase() === 'æ') {
+            countÆ++;
+        }
+        if(letter.toLowerCase() === 'ø') {
```
[the remark this answers is not in the document]
Comment (TITHI1244) [reply javascript/javascript2/week1/count_letters.js:9]: Yes :)

File: javascript/javascript2/week1/spirit_animals_vs2.html [hunk 3]
```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <script src="newAnimals.js"></script>
```
[the remark this answers is not in the document]
Comment (TITHI1244) [reply javascript/javascript2/week1/spirit_animals_vs2.html:9]: Yes, I changes the file name later but forgot to change the link :)

File: javascript/javascript2/week1/spirit_animal.js [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+// create a button element and append it to the body
+const btnElement = document.createElement("button");
+const btnText = document.createTextNode("Click me to find out!");
+btnElement.appendChild(btnText);
+document.body.appendChild(btnElement);
+
+// create a h3 tag and append it to the body, set the text to a primary value
+const h3Element = document.createElement("h3");
+h3Element.innerHTML = "Excited to find out??";
+document.body.appendChild(h3Element);
```
[the remark this answers is not in the document]
Comment (TITHI1244) [reply javascript/javascript2/week1/spirit_animal.js:17]: Ok will change that. I actually thought after reading the task description that we should add the elements from .js.

File: javascript/javascript2/week1/spirit_animals_vs2.js [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+// create a button element and append it to the body
+const btnElement = document.createElement("button");
+const btnText = document.createTextNode("Click me to find out!");
+btnElement.appendChild(btnText);
+document.body.appendChild(btnElement);
+
+// create a h3 element and append it to the body, set the text to a primary value
+const h3Element = document.createElement("h3");
+h3Element.innerHTML = "Excited to find out??";
+document.body.appendChild(h3Element);
```
[the remark this answers is not in the document]
Comment (TITHI1244) [reply javascript/javascript2/week1/spirit_animals_vs2.js:66]: Yes.

File: javascript/javascript2/week1/spirit_animals_vs2.js [hunk 6]
```diff
@@ -0,0 +1,105 @@
...
+formElement.onclick = function() {
+    userChoice = document.querySelector('input[name = user_choice]:checked').value;
+    if(userChoice === "Click the button") {
+        btnElement.addEventListener("click", showTheAnimal);
+    } else if(userChoice === "Hover over the input field") {
+        document.getElementById("user_input").addEventListener("mouseenter", showTheAnimal);
+    } else if(userChoice === "While I type") {
+        document.getElementById("user_input").addEventListener("keydown", showTheAnimal);
+    } 
+}
```
[the remark this answers is not in the document]
Comment (TITHI1244) [reply javascript/javascript2/week1/spirit_animals_vs2.js:105]: Yes, changed the code. Thanks :)

File: javascript/javascript2/week1/main.js [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+        liElement.appendChild(brElement);
+
+        // create another span element and append to the li, assign the product rating to its text
+        const spanElement2 = document.createElement("span");
+        spanElement2.innerHTML = `Rating: ${product.rating}`;
+        liElement.appendChild(spanElement2);
+    }
+}
+
+renderProducts(products);
```
[the remark this answers is not in the document]
Comment (TITHI1244) [reply javascript/javascript2/week1/main.js:37]: Thanks :)

File: javascript/javascript2/week1/main.js [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+        liElement.appendChild(h2Element);
+
+        // create a span element and append to the li, assign the product price to its text
+        const spanElement = document.createElement("span");
+        spanElement.innerHTML = `price: ${product.price}`;
+        liElement.appendChild(spanElement);
+
+        // create a br element and append to the li to separate the lines
+        const brElement = document.createElement("br");
+        liElement.appendChild(brElement);
```
[the remark this answers is not in the document]
Comment (TITHI1244) [reply javascript/javascript2/week1/main.js:28]: Yes. I saw the given html code in sandbox and there these elements were used, so I used these span and br to match that code :) Will change that.

**Pull request merged**